frontend/src/app/users/users-profile/users-profile.component.html

```diff
@@ -0,0 +1,27 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<h2>Profile of {{ userName }}</h2>
+
+<div *ngIf="userId !== currentUser?.id">
+  Common Projects
+
+  <ng-container
+    *ngIf="(projectService.projects$ | async)?.length; else elseBlock"
+  >
+    <mat-card
+      *ngFor="let project of projectService.projects$ | async"
+      class="!ml-0 cursor-pointer"
+      [routerLink]="['/project', project.slug]"
+    >
+      <mat-card-content>
+        <b>{{ project.name }}</b
+        ><br />
+        {{ project.description }}
+      </mat-card-content>
+    </mat-card>
+  </ng-container>
+  <ng-template #elseBlock>You do not have any common projects.</ng-template>
```

Comment:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/f15c90bc-2d6d-488e-8091-0f61e38f97c8)

A new line is missing here between Projects and you